File: 251231/몫 출력하기/print-share.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim();
const numberList = input.split('\n').map(Number);
let [index, count] = [0, 0];

while (true) {
    if (numberList[index] % 2 === 0) {
        count += 1;
        console.log(numberList[index] / 2);
    }

    if (count === 3) {
        break;
    }

    index += 1;
}

File: 260312/몫 출력하기/print-share.py
import sys

input = sys.stdin.readline

# 선언부

# 입력부

# 호출부
count = 0
while True:
    i = int(input())
    if i % 2 != 0:
        continue
    else:
        print(i // 2)
        count += 1

    if count == 3:
        break